sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <modal-dialog-base :showDialog="showDialog" title="StudentAid BC Declaration">
+    <template #content>
+      <p>
+        Please review and accept the StudentAid BC declaration to allow the
+        institution to make proposed changes to your application (on your
+        behalf).
+      </p>
+      <content-group-info class="my-3">
+        <v-row
+          ><v-col
+            ><h4 class="label-bold color-blue">
+              StudentAid BC Declaration
+            </h4></v-col
+          ></v-row
+        >
+        <v-row
+          ><v-col
+            ><h4 class="label-bold">I. I. I/we understand that:</h4>
+          </v-col></v-row
+        ><v-list bg-color="#f5f6f7" class="mb-5">
+          <v-list-item>
+            1. The student will have access to information provided in this
+            appendix;
+          </v-list-item>
+          <v-list-item>
+            2. The student's school will have access to information provided in
+            this appendix;
+          </v-list-item>
+          <v-list-item>
+            3. The information in this appendix is subject to audit,
+            investigation and verification as defined in the current program
+            year's StudentAid BC Policy Manual
+          </v-list-item>
+        </v-list>
+        <v-row
+          ><v-col
+            ><h4 class="label-bold">Collection and use of information</h4>
+          </v-col></v-row
+        >
+        <v-row
+          ><v-col
+            >The information included in this form and authorized above is
+            collected and managed in accordance with sections 26(c) and 26(e) of
+            the Freedom of Information and Protection of Privacy Act, R.S.B.C.
+            1996, c. 165, and under the authority of the Canada Student
+            Financial Assistance Act, R.S.C. 1994, Chapter C-28 and StudentAid
+            BC. The information provided will be used to determine eligibility
+            for a benefit through</v-col
+          ></v-row
+        > </content-group-info
+      ><v-checkbox
+        ref="declarationCheckbox"
```

Comment:
I belive the ref should be added for the `v-form`

Reply:
It was redundant. I have removed it.